src/main/java/Application.java

```diff
@@ -0,0 +1,34 @@
+import domain.Race;
+import view.InputView;
+import view.ResultView;
+
+public class Application {
+
+    private static Race race;
+    private static int round;
+    public static void main(String[] args) {
+        while(true){
+            if (getValiedNames()){
+                break; //5글자를 넘는 이름을 입력받는 경우 재시도
+            }
+        }
```

Comment:
재시도 로직 ! 저도 생각도 못해 본 부분이네요 👀 
 
다음 같은 방법이 있을 것 같아요.
1. 메소드를 세세하게 더 쪼갠다. (이 부분은 작성해보지 못 했지만, 코드가 쉽지 않아질 것 같다는 생각이 드네요.)
2. do-while문을 사용한다. (이렇게 되면 뎁스가 1이하가 되고, 재시도로직도 잘 작동할 것 같아요!)
```suggestion
        String carNames;
        do{
            carNames = InputView.getCarNames();
        } while(!getValiedNames(carNames));
```

Reply:
do-while이 있었네요! 맨날 for, while만 쓰고 do-while은 정말 쓰는 케이스가 적어서 생각을 못했습니다 👍